kingpin/schema.py

```diff
@@ -20,7 +20,7 @@
 
 
 SCHEMA_1_0 = {
-    'type': 'object',
+    'type': ['object', 'array'],
```

Comment:
Lets chat about this. I think that we need an "outer" schema here that can handle either a list of SCHEMA_1_0 objects, or just a single SCHEMA_1_0 object.